### ISSUE TITLE
"Warning: Accessing non-existent property" after updating to node 14.X

### ISSUE THREAD

**Issue by lampewebdev** [nestjs/nest-cli#707]
## I'm submitting a...
<pre><code>
[ ] Regression 
[x] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior
When running 'nest' I get the following output

```
micha@BLADE:~/websockets-tutorial$ nest
Usage: nest <command> [options]

Options:
  -v, --version                                   Output the current version.
  -h, --help                                      Output usage information.

Commands:
  new|n [options] [name]                          Generate Nest application.
  build [options] [app]                           Build Nest application.
  start [options] [app]                           Run Nest application.
  info|i                                          Display Nest project details.
  update|u [options]                              Update Nest dependencies.
  add [options] <library>                         Adds support for an external library to your project.
  generate|g [options] <schematic> [name] [path]  Generate a Nest element.
    Available schematics:
      ┌───────────────┬─────────────┐
      │ name          │ alias       │
      │ application   │ application │
      │ class         │ cl          │
      │ configuration │ config      │
      │ controller    │ co          │
      │ decorator     │ d           │
      │ filter        │ f           │
      │ gateway       │ ga          │
      │ guard         │ gu          │
      │ interceptor   │ in          │
      │ interface     │ interface   │
      │ middleware    │ mi          │
      │ module        │ mo          │
      │ pipe          │ pi          │
      │ provider      │ pr          │
      │ resolver      │ r           │
      │ service       │ s           │
      │ library       │ lib         │
      │ sub-app       │ app         │
      └───────────────┴─────────────┘
(node:813) Warning: Accessing non-existent property 'cat' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
(node:813) Warning: Accessing non-existent property 'cd' of module exports inside circular dependency
(node:813) Warning: Accessing non-existent property 'chmod' of module exports inside circular dependency
(node:813) Warning: Accessing non-existent property 'cp' of module exports inside circular dependency
(node:813) Warning: Accessing non-existent property 'dirs' of module exports inside circular dependency
(node:813) Warning: Accessing non-existent property 'pushd' of module exports inside circular dependency
(node:813) Warning: Accessing non-existent property 'popd' of module exports inside circular dependency
(node:813) Warning: Accessing non-existent property 'echo' of module exports inside circular dependency
(node:813) Warning: Accessing non-existent property 'tempdir' of module exports inside circular dependency
(node:813) Warning: Accessing non-existent property 'pwd' of module exports inside circular dependency
(node:813) Warning: Accessing non-existent property 'exec' of module exports inside circular dependency
(node:813) Warning: Accessing non-existent property 'ls' of module exports inside circular dependency
(node:813) Warning: Accessing non-existent property 'find' of module exports inside circular dependency
(node:813) Warning: Accessing non-existent property 'grep' of module exports inside circular dependency
(node:813) Warning: Accessing non-existent property 'head' of module exports inside circular dependency
(node:813) Warning: Accessing non-existent property 'ln' of module exports inside circular dependency
(node:813) Warning: Accessing non-existent property 'mkdir' of module exports inside circular dependency
(node:813) Warning: Accessing non-existent property 'rm' of module exports inside circular dependency
(node:813) Warning: Accessing non-existent property 'mv' of module exports inside circular dependency
(node:813) Warning: Accessing non-existent property 'sed' of module exports inside circular dependency
(node:813) Warning: Accessing non-existent property 'set' of module exports inside circular dependency
(node:813) Warning: Accessing non-existent property 'sort' of module exports inside circular dependency
(node:813) Warning: Accessing non-existent property 'tail' of module exports inside circular dependency
(node:813) Warning: Accessing non-existent property 'test' of module exports inside circular dependency
(node:813) Warning: Accessing non-existent property 'to' of module exports inside circular dependency
(node:813) Warning: Accessing non-existent property 'toEnd' of module exports inside circular dependency
(node:813) Warning: Accessing non-existent property 'touch' of module exports inside circular dependency
(node:813) Warning: Accessing non-existent property 'uniq' of module exports inside circular dependency
(node:813) Warning: Accessing non-existent property 'which' of module exports inside circular dependency
```

## Expected behaviour
I don't want to get that output

## Minimal reproduction of the problem with instructions
- npm i -g @nest/cli
- nest

## What is the motivation/use case for changing the behaviour?
Cleaner output

## Environment

```
[System Information]
OS Version     : Linux 4.19
NodeJS Version : v14.1.0
(node:820) Warning: Accessing non-existent property 'cat' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
(node:820) Warning: Accessing non-existent property 'cd' of module exports inside circular dependency
(node:820) Warning: Accessing non-existent property 'chmod' of module exports inside circular dependency
(node:820) Warning: Accessing non-existent property 'cp' of module exports inside circular dependency
(node:820) Warning: Accessing non-existent property 'dirs' of module exports inside circular dependency
(node:820) Warning: Accessing non-existent property 'pushd' of module exports inside circular dependency
(node:820) Warning: Accessing non-existent property 'popd' of module exports inside circular dependency
(node:820) Warning: Accessing non-existent property 'echo' of module exports inside circular dependency
(node:820) Warning: Accessing non-existent property 'tempdir' of module exports inside circular dependency
(node:820) Warning: Accessing non-existent property 'pwd' of module exports inside circular dependency
(node:820) Warning: Accessing non-existent property 'exec' of module exports inside circular dependency
(node:820) Warning: Accessing non-existent property 'ls' of module exports inside circular dependency
(node:820) Warning: Accessing non-existent property 'find' of module exports inside circular dependency
(node:820) Warning: Accessing non-existent property 'grep' of module exports inside circular dependency
(node:820) Warning: Accessing non-existent property 'head' of module exports inside circular dependency
(node:820) Warning: Accessing non-existent property 'ln' of module exports inside circular dependency
(node:820) Warning: Accessing non-existent property 'mkdir' of module exports inside circular dependency
(node:820) Warning: Accessing non-existent property 'rm' of module exports inside circular dependency
(node:820) Warning: Accessing non-existent property 'mv' of module exports inside circular dependency
(node:820) Warning: Accessing non-existent property 'sed' of module exports inside circular dependency
(node:820) Warning: Accessing non-existent property 'set' of module exports inside circular dependency
(node:820) Warning: Accessing non-existent property 'sort' of module exports inside circular dependency
(node:820) Warning: Accessing non-existent property 'tail' of module exports inside circular dependency
(node:820) Warning: Accessing non-existent property 'test' of module exports inside circular dependency
(node:820) Warning: Accessing non-existent property 'to' of module exports inside circular dependency
(node:820) Warning: Accessing non-existent property 'toEnd' of module exports inside circular dependency
(node:820) Warning: Accessing non-existent property 'touch' of module exports inside circular dependency
(node:820) Warning: Accessing non-existent property 'uniq' of module exports inside circular dependency
(node:820) Warning: Accessing non-existent property 'which' of module exports inside circular dependency
NPM Version    : 6.14.4

[Nest CLI]
Nest CLI Version : 7.1.4
```

**Comment by peterjdrb** [comment]
I also get the same output when executing "nest new test"

<pre><code>
[System Information]
OS Version     : Windows 10 Pro Version 1909
NodeJS Version : v14.1.0

nest new test
⚡  We will scaffold your app in a few seconds..

(node:13868) Warning: Accessing non-existent property 'cat' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
(node:13868) Warning: Accessing non-existent property 'cd' of module exports inside circular dependency
(node:13868) Warning: Accessing non-existent property 'chmod' of module exports inside circular dependency
(node:13868) Warning: Accessing non-existent property 'cp' of module exports inside circular dependency
(node:13868) Warning: Accessing non-existent property 'dirs' of module exports inside circular dependency
(node:13868) Warning: Accessing non-existent property 'pushd' of module exports inside circular dependency
(node:13868) Warning: Accessing non-existent property 'popd' of module exports inside circular dependency
(node:13868) Warning: Accessing non-existent property 'echo' of module exports inside circular dependency
(node:13868) Warning: Accessing non-existent property 'tempdir' of module exports inside circular dependency
(node:13868) Warning: Accessing non-existent property 'pwd' of module exports inside circular dependency
(node:13868) Warning: Accessing non-existent property 'exec' of module exports inside circular dependency
(node:13868) Warning: Accessing non-existent property 'ls' of module exports inside circular dependency
(node:13868) Warning: Accessing non-existent property 'find' of module exports inside circular dependency
(node:13868) Warning: Accessing non-existent property 'grep' of module exports inside circular dependency
(node:13868) Warning: Accessing non-existent property 'head' of module exports inside circular dependency
(node:13868) Warning: Accessing non-existent property 'ln' of module exports inside circular dependency
(node:13868) Warning: Accessing non-existent property 'mkdir' of module exports inside circular dependency
(node:13868) Warning: Accessing non-existent property 'rm' of module exports inside circular dependency
(node:13868) Warning: Accessing non-existent property 'mv' of module exports inside circular dependency
(node:13868) Warning: Accessing non-existent property 'sed' of module exports inside circular dependency
(node:13868) Warning: Accessing non-existent property 'set' of module exports inside circular dependency
(node:13868) Warning: Accessing non-existent property 'sort' of module exports inside circular dependency
(node:13868) Warning: Accessing non-existent property 'tail' of module exports inside circular dependency
(node:13868) Warning: Accessing non-existent property 'test' of module exports inside circular dependency
(node:13868) Warning: Accessing non-existent property 'to' of module exports inside circular dependency
(node:13868) Warning: Accessing non-existent property 'toEnd' of module exports inside circular dependency
(node:13868) Warning: Accessing non-existent property 'touch' of module exports inside circular dependency
(node:13868) Warning: Accessing non-existent property 'uniq' of module exports inside circular dependency
(node:13868) Warning: Accessing non-existent property 'which' of module exports inside circular dependency
CREATE test/.eslintrc.js (599 bytes)
CREATE test/.prettierrc (51 bytes)
CREATE test/nest-cli.json (64 bytes)
CREATE test/package.json (1892 bytes)
CREATE test/README.md (3370 bytes)
CREATE test/tsconfig.build.json (97 bytes)
CREATE test/tsconfig.json (336 bytes)
CREATE test/src/app.controller.spec.ts (617 bytes)
CREATE test/src/app.controller.ts (274 bytes)
CREATE test/src/app.module.ts (249 bytes)
CREATE test/src/app.service.ts (142 bytes)
CREATE test/src/main.ts (208 bytes)
CREATE test/test/app.e2e-spec.ts (630 bytes)
CREATE test/test/jest-e2e.json (183 bytes)

? Which package manager would you ❤️  to use? npm
√ Installation in progress... ☕

�  Successfully created project test
�  Get started with the following commands:

$ cd test
$ npm run start


                          Thanks for installing Nest �
                 Please consider donating to our open collective
                        to help us maintain this package.


               �  Donate: https://opencollective.com/nest
</code></pre>

**Comment by kamilmysliwiec** [comment]
Fixed in 7.1.5
